### ISSUE TITLE
Item slicing on latest forge

### ISSUE THREAD

**Issue by tmzasz** [Chicken-Bones/NotEnoughItems#192]
ive noticed a strange graphical glitch on forge 1.7.10-10.13.3.1370-1.7.10
when you open the nei interface or push R on an item this occurs:
![with nei smp](https://cloud.githubusercontent.com/assets/7916657/7285732/c04263e2-e914-11e4-9765-8562da7586e5.png)
![with nei single player](https://cloud.githubusercontent.com/assets/7916657/7285731/c04186f2-e914-11e4-8c91-1a5dca0a0032.png)
but when i remove nei ( and translocators as it apparently needs nei ) this is what happens ( what should happen )
![without nei installed](https://cloud.githubusercontent.com/assets/7916657/7285753/e752628e-e914-11e4-9730-93a47a83da99.png)
ive tested this in both ssp and smp its consistent

**Comment by GirafiStudios** [comment]
Have you tried without that texturepack?

**Comment by tmzasz** [comment]
yes ive tried 3 different including default and it does the same thing i noticed it also takes a complete relaunch of MC to resolve it usually though 2 or 3 times it has resolved itself somewhat just to return
the 2 packs other than default i have used are soratex fanver and faithful

**Comment by tmzasz** [comment]
ok a little more testing on the default texture pack some pages in nei can randomly cause it but page one in large gui mode causes it consistently some vanilia block/item must be triggering the rendering issue perhaps
![further testing](https://cloud.githubusercontent.com/assets/7916657/7291324/b0339f78-e95c-11e4-8afd-04980445d245.png)
